policyengine_us/variables/gov/states/ky/tax/income/credits/family_size_credit/ky_family_size_tax_credit_rate.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class ky_family_size_tax_credit_rate(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Kentucky family size tax credit rate"
+    unit = "/1"
+    definition_period = YEAR
+    reference = (
+        "https://apps.legislature.ky.gov/law/statutes/statute.aspx?id=49188"
+    )
+    defined_for = StateCode.KY
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("ky_modified_agi", period)
+        fpg = parameters(period).gov.hhs.fpg
+        state_group = "CONTIGUOUS_US"
```

Comment:
To reduce hard-coding
```suggestion
        # This will be CONTIGUOUS_US for Kentucky.
        state_group = tax_unit.household("state_group", period)
```